UnityProject/Assets/Scripts/UI/OxygenButton.cs

```diff
@@ -42,7 +42,8 @@ public void OxygenSelect()
 			return;
 		}
 
-		SoundManager.Play("Click01");
+		// JESTER
+		//SoundManager.Play("Click01");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)